templates/base_index.html

```diff
@@ -51,6 +51,12 @@
     <meta name="twitter:image" content="{% if 'http' not in self.meta_image() %}https://assets.ubuntu.com/v1/{% endif %}{{ self.meta_image() }}">
     <meta property="og:image" content="{% if 'http' not in self.meta_image() %}https://assets.ubuntu.com/v1/{% endif %}{{ self.meta_image() }}">
     {% endif %}
+    
+    {% set current_path =  url_for(request.endpoint, **request.view_args)  %}
+    {% if "/careers" in current_path %}
```

Comment:
It'll only be true if the path starts with `/careers`. But with what we have now if we got a path that looks like `/blahblahblah/careers...` it would also be true.

Feel free to ignore it, what are the odds that will happen anyway.

Reply:
I would assume that that's exactly the behavior we want. Even if the entire careers bubble gets moved within another index (which I agree is quite unlikely) the meta-image should not change unless requested. eg: `/company/careers/job_post_id`